share/wake/lib/system/job.wake

```diff
@@ -320,6 +320,11 @@ export def runJob p = match p
         def _ = wait (\_ badfinish job error) (badlaunch job error)
         job
 
+# Job => String => String => Unit
+# Set the value of a tag on a Job
+# This is useful for post-build reflection into the database
+export def setJobTag job key value = prim "job_tag"
+
```

Comment:
So this is defined on `Job` right? What happens if you set the same key multiple times, overwrite?

Reply:
No. First time the tag is set, that's what sticks.

Hmm. That's probably bad, actually, in case you modify the wake code but the job is not re-executed. I'll change the behaviour to overwrite.

Reply:
Fixed.